Segment-Mixpanel.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0.0'
+  s.dependency 'Analytics', '~> 3'
```

Comment:
I'm sorry, I know how the versioning works but I misread your comment in relation to the diff that said `s.dependency 'Analytics', '~> 3'`. My apologies.